### DISCUSSION TITLE
Beaconstation_KopraKoht oxygen generator is not wired

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13995] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Beaconstation_KopraKoht oxygen generator is not wired

### Reproduction steps

Beaconstation_KopraKoht oxygen generator is not wired

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for the report, a push has been fixed and will find its way to a beacon station near you soon, closing the discussion.
